Makefile

```diff
@@ -1,11 +0,0 @@
-
-all:
```

Comment:
Worth keeping the makefile build, if we're submitting this upstream? That means people can still do a simple build without needing to go through cmake - the code is minimal so the maintenance effort shouldn't be too bad.